spec/example.spec.js

```diff
@@ -0,0 +1,15 @@
+describe('Test that the tests are running', function(){
```

Comment:
If we still want to keep this file, I'd do two things:

1. Put it in a "test" folder, rather than "spec". 

2. I'd like to stick with the Google convention, which is to name test files the same as their source, with a "_test" subscript (e.g. my-source-file.js -> my-source-file_test.js).

Reply:
Ok, I changed the name to fit the convention